openvpn/start.sh

```diff
@@ -59,19 +59,43 @@ fi
 echo "Using OpenVPN provider: ${VPN_PROVIDER^^}"
 
 if [[ -n $OPENVPN_CONFIG_URL ]]; then
-  echo "Found URL to OpenVPN config, will download it."
+  echo "Found URL to single OpenVPN config, will download and use it."
   CHOSEN_OPENVPN_CONFIG=$VPN_PROVIDER_HOME/downloaded_config.ovpn
   curl -o "$CHOSEN_OPENVPN_CONFIG" -sSL "$OPENVPN_CONFIG_URL"
-  MODIFY_CHOSEN_CONFIG=yeah
-elif [[ -x $VPN_PROVIDER_HOME/configure-openvpn.sh ]]; then
-  echo "Provider $OPENVPN_PROVIDER has a custom setup script, executing it"
-  # Preserve $PWD in case it changes when sourcing the script
-  pushd -n "$PWD" > /dev/null
-  # shellcheck source=/dev/null
-  . "$VPN_PROVIDER_HOME"/configure-openvpn.sh
-  # Restore previous PWD
-  popd > /dev/null
-  MODIFY_CHOSEN_CONFIG=yeah
+fi
+
+if [[ -z ${CHOSEN_OPENVPN_CONFIG} ]]; then
```

Comment:
missing double quotes

Reply:
I know that openvpn/start.sh has a bit of syntactical issues. Have run shellcheck on it and I'll do a fixup later. Don't want to bundle it with this change in case I have to roll back.

Either way. For the `-z` empty checks it doesn't seem to mind about skipping the quotes. Maybe because if the string is empty there will be no word splitting? And if it's not empty it doesn't really matter if the string is split because it's not empty anyways?

Reply:
I stand corrected. The need to surround parameters for `test -z` must be an anachronism.